HW1/P2/P2a.py

```diff
@@ -1,3 +1,14 @@
 from P2 import *
+import findspark
+findspark.init()
+import pyspark
+sc = pyspark.SparkContext(appName="Spark1")
+my_rdd = sc.parallelize(xrange(2000),10)
+rdd = my_rdd.cartesian(my_rdd)
+ans = rdd.map(lambda x: [x,mandelbrot((x[1]/500.0-2),(x[0]/500.0-2))])
+draw_image(ans)
 
-# Your code here
+ans1 = sum_values_for_partitions(ans)
+
+plt.hist(ans1.collect())
```

Comment:
You should really make this histogram on a log-log scale. If you do that, you will see that work per partition varies over 3 orders of magnitude. You can't see that at all right now...it looks like all partitions complete O(10^7) iterations which is ok. A linear histogram does not  illustrate the disparity appropriately in my opinion.